src/utils/rules-proxy.js

```diff
@@ -17,19 +17,25 @@ const parseFile = async function (filePath) {
   }
 
   const parser = path.basename(filePath) === '_redirects' ? parseRedirectsFormat : parseNetlifyConfig
-  const { success, errors } = await parser(filePath)
-  if (errors.length !== 0) {
-    console.error(`${NETLIFYDEVWARN} Warnings while parsing ${path.basename(filePath)} file:`)
-    errors.forEach(({ lineNum, line, reason }) => {
-      console.error(`  ${lineNum}: ${line} -- ${reason}`)
-    })
+  try {
+    const rules = await parser(filePath)
+    return rules.map(normalizeRule)
+  } catch (error) {
+    console.error(`${NETLIFYDEVWARN} Warnings while parsing ${path.basename(filePath)} file:
+${error.message}`)
+    return []
   }
-  return success
 }
 
 const parseRules = async function (configFiles) {
-  const results = await Promise.all(configFiles.map(parseFile))
-  return [].concat(...results)
+  const rules = await Promise.all(configFiles.map(parseFile))
+  return rules.flat()
```

Comment:
I think this is only supported on Node.js >= 11.

However it did seem to work when I tested it on Node.js 10, I think due to https://github.com/netlify/zip-it-and-ship-it/blob/08f2a5d49a78e4424bc7478aa3cc8c98d21c5ee7/src/main.js#L3

Reply:
Yes, you're correct, this was introduced by Node `11.0.0` (I just checked).
We should return to `[].concat()`, sorry about this :+1: